otx/algorithms/classification/tasks/openvino.py

```diff
@@ -0,0 +1,374 @@
+from otx.api.entities.annotation import AnnotationSceneEntity
+from otx.api.entities.datasets import DatasetEntity
+from otx.api.entities.inference_parameters import (
+    InferenceParameters,
+    default_progress_callback,
+)
+from otx.api.entities.label_schema import LabelSchemaEntity
+from otx.api.entities.metadata import FloatMetadata, FloatType
+from otx.api.entities.model import (
+    ModelEntity,
+    ModelFormat,
+    ModelOptimizationType,
+    ModelPrecision,
+    OptimizationMethod,
+)
+from otx.api.entities.optimization_parameters import OptimizationParameters
+from otx.api.entities.result_media import ResultMediaEntity
+from otx.api.entities.resultset import ResultSetEntity
+from otx.api.entities.task_environment import TaskEnvironment
+from otx.api.entities.tensor import TensorEntity
+from otx.api.serialization.label_mapper import LabelSchemaMapper, label_schema_to_bytes
+from otx.api.usecases.evaluation.metrics_helper import MetricsHelper
+from otx.api.usecases.exportable_code import demo
+from otx.api.usecases.exportable_code.inference import BaseInferencer
+from otx.api.usecases.exportable_code.prediction_to_annotation_converter import (
+    ClassificationToAnnotationConverter,
+)
+from otx.api.usecases.tasks.interfaces.deployment_interface import IDeploymentTask
+from otx.api.usecases.tasks.interfaces.evaluate_interface import IEvaluationTask
+from otx.api.usecases.tasks.interfaces.inference_interface import IInferenceTask
+from otx.api.usecases.tasks.interfaces.optimization_interface import (
+    IOptimizationTask,
+    OptimizationType,
+)
+from otx.api.utils.argument_checks import (
+    DatasetParamTypeCheck,
+    check_input_parameters_type,
+)
+
+try:
+    from openvino.model_zoo.model_api.adapters import OpenvinoAdapter, create_core
+    from openvino.model_zoo.model_api.models import Model
+except ImportError:
+    import warnings
+
+    warnings.warn("ModelAPI was not found.")
+
+logger = logging.getLogger(__name__)
+
+
+class ClassificationOpenVINOInferencer(BaseInferencer):
+    """ClassificationOpenVINOInferencer class in OpenVINO task."""
+
+    @check_input_parameters_type()
+    def __init__(
+        self,
+        hparams: ClassificationConfig,
+        label_schema: LabelSchemaEntity,
+        model_file: Union[str, bytes],
+        weight_file: Union[str, bytes, None] = None,
+        device: str = "CPU",
+        num_requests: int = 1,
+    ):  # pylint: disable=unused-argument
+        """Inferencer implementation for OTXDetection using OpenVINO backend.
+
+        :param model: Path to model to load, `.xml`, `.bin` or `.onnx` file.
+        :param hparams: Hyper parameters that the model should use.
+        :param num_requests: Maximum number of requests that the inferencer can make.
+            Good value is the number of available cores. Defaults to 1.
+        :param device: Device to run inference on, such as CPU, GPU or MYRIAD. Defaults to "CPU".
```

Comment:
@goodsong81 I think many files in mpa has to revisit about style. I think it would be better to fix it at once. I will add it FIXME annotation above this.